sympy/calculus/finite_diff.py

```diff
@@ -38,8 +38,8 @@ def finite_diff_weights(order, x_list, x0=S.One):
         0 corresponds to interpolation.
     x_list: sequence
         Sequence of (unique) values for the independent variable.
-        It is usefull (but not necessary) to order ``x_list`` from
-        nearest to farest from ``x0``; see examples below.
+        It is useful (but not necessary) to order ``x_list`` from
+        nearest to farthest from ``x0``; see examples below.
```

Comment:
I would recommend using "furthest" instead of "farthest" (the latter being used more often when referring to physical distances).